landscape.rst

```diff
@@ -31,9 +31,26 @@ There are 3 versions of Landscape:
 Getting started with Landscape
 ------------------------------
 
+Landscape SaaS
+~~~~~~~~~~~~~~
 Before you can access your Landscape SaaS account for the first time, you need to follow the steps under :doc:`Account set-up <account-setup>`.
 
-For customers using self-hosted Landscape, your ``licence.txt`` file is available in your Landscape SaaS account. You will need to re-download and apply new ``licence.txt`` files every time you purchase new Ubuntu Pro licences and after every renewal.
+Self-hosted Landscape
+~~~~~~~~~~~~~~~~~~~~~
+
+We currently have two licensing mechanisms for self hosted Landscape: the new, Pro client method and the old, licence.txt file method.
```

Comment:
Change adding link (as discussed)
After you have set up self-hosted Landscape server (hyperlink to Landscape docs) there are two licensing mechanisms: the new, Pro client method and the old, licence.txt file method.